content/api/versions/v0.8/rest-sdk/message.textile

```diff
@@ -0,0 +1,199 @@
+---
+title: Message
+section: api
+index: 40
+jump_to:
```

Comment:
I think this one is missing the language selector options.

Reply:
Added.